tests/integration/cattletest/core/test_authorization.py

```diff
@@ -13,19 +13,20 @@ def test_user_client_access(clients):
         'superadmin': 145,
         'service': 93,
         'project': 71,
-        'admin': 97,
+        'admin': 98,
         'agent': 8,
-        'user': 75,
+        'user': 76,
         'agentRegister': 4,
         'test': 140,
-        'readAdmin': 97,
+        'readAdmin': 98,
         'token': 2,
-        'superadmin': 151,
-        'service': 97,
-        'project': 75,
+        'superadmin': 152,
+        'service': 98,
+        'project': 76,
     }
     for tuple in clients.items():
-        assert typesLen[tuple[0]] == len(tuple[1].schema.types.items())
+        assert tuple[0] == tuple[0] and \
```

Comment:
a better alternative would be something like:
```
assert typesLen[tuple[0]] == len(tuple[1].schema.types.items()), 'Discrepency in account %s' % tuple[0]
```
...you can add a string after an assert and thats the message that will be displayed if that assert failes

also, generally speaking, i hate this entire test function